zds/tutorialv2/models/models_database.py

```diff
@@ -135,8 +141,12 @@ def textual_type(self):
         """
         if self.is_article():
             return _(u"L'Article")
-        else:
+        elif self.is_opinion():
+            return _(u"Le Billet")
+        elif self.is_tutorial():
             return _(u"Le Tutoriel")
+        else:
+            return _(u"Le Contenu")
```

Comment:
pourquoi laisser comme ça?

- parce que finalement, le fait de ne pas faire partie de la liste des types que zds utilise n'est pas un comportement exceptionnel; On peut tout à fait imaginer un fork qui ajoute un nouveau type de contenu à la volée. Bon maintenant, pour rendre ça viable il faudrait revoir le code mais comme *if ain't broken don't fix it* on le fait pas.
- parce qu'on ne sait pas où ça cassera
- parce que clairement là on est en programmation défensive, la chance pour que "Contenu" arrive un jour est faible
- parce que justement, quand on devra développer une extension au module, passer son temps à débugger les exceptions qui pop de ci, de là, c'est contr indiqué. De ce que j'ai vu sur irc, gustavi, qui est pourtant un développeur très actif sur zds a eu du mal à rentrer dans le module alors qu'avec pierre on pensait l'avoir gravement simplifier. Ajouter une nouvelle barrière, ça serait contreproductif.